PostProcessing/Runtime/PostProcessLayer.cs

```diff
@@ -244,7 +244,7 @@ void InitLegacy()
         [ImageEffectUsesCommandBuffer]
         void OnRenderImage(RenderTexture src, RenderTexture dst)
         {
-            if (finalBlitToCameraTarget)
+            if (finalBlitToCameraTarget && (m_Camera.allowDynamicResolution && ScalableBufferManager.heightScaleFactor == 1.0 && ScalableBufferManager.widthScaleFactor == 1.0))
```

Comment:
This looks wrong.
Shouldn't this condition match line 610?
Maybe extract the dynamic resolution part into a function?

Reply:
Good catch. I'll switch it to some function for ease of reading too.